cliquet/tests/test_views_hello.py

```diff
@@ -46,3 +46,12 @@ def test_redirect_to_version(self):
                              'http://localhost/v0/mushrooms')
         finally:
             self.app.RequestClass = original_request_class
+
+    def test_do_not_redirect_to_version(self):
```

Comment:
If we consider the name of test as a spec (see http://blog.mathieu-leplatre.info/your-tests-as-your-specs.html), then the name of this test would be "do not redirect to version if disabled in settings"